git-crypt-setup.iss

```diff
@@ -0,0 +1,42 @@
+; Script generated by the Inno Setup Script Wizard.
+; SEE THE DOCUMENTATION FOR DETAILS ON CREATING INNO SETUP SCRIPT FILES!
+
+#define MyAppName "Git Crypt for Windows"
+#define MyAppVersion "0.6.0"
+#define MyAppPublisher "SIL International"
+#define MyAppExeName "git-crypt.exe"
+
+[Setup]
+; NOTE: The value of AppId uniquely identifies this application. Do not use the same AppId value in installers for other applications.
+; (To generate a new GUID, click Tools | Generate GUID inside the IDE.)
+AppId={{C805A279-D06E-464C-A7DB-D4265CC1AC2D}
+AppName={#MyAppName}
+AppVersion={#MyAppVersion}
+;AppVerName={#MyAppName} {#MyAppVersion}
+AppPublisher={#MyAppPublisher}
+DefaultDirName=C:\Program Files\Git\cmd
+DefaultGroupName={#MyAppName}
+DisableProgramGroupPage=yes
+LicenseFile=LICENSE
+; Uncomment the following line to run in non administrative install mode (install for current user only.)
+;PrivilegesRequired=lowest
+OutputDir=Installer
+OutputBaseFilename=git-crypt-setup
+Compression=lzma
+SolidCompression=yes
+WizardStyle=modern
+
+[Languages]
+Name: "english"; MessagesFile: "compiler:Default.isl"
+
+[Files]
+Source: "git-crypt\{#MyAppExeName}"; DestDir: "{app}"; Flags: ignoreversion
+Source: "C:\dev\msys64\mingw64\bin\libcrypto-1_1-x64.dll"; DestDir: "{app}"; Flags: ignoreversion
```

Comment:
I'm not sure about these source locations, is dev the default install location for msys64?

Reply:
C:\dev\msys64 is the location included in the instructions I referenced for installing `g++` (https://github.com/orlp/dev-on-windows/wiki/Installing-GCC--&-MSYS2).  The instructions state:
> Make sure to set the install directory to C:\dev\msys64

I could change the setup script to prompt for the location of the msys64 directory instead.

Reply:
No, it is fine. I didn't dig into those instructions.